agot-bg-game-server/src/common/ingame-game-state/westeros-game-state/wildlings-attack-game-state/WildlingsAttackGameState.ts

```diff
@@ -152,7 +153,8 @@ export default class WildlingsAttackGameState extends GameState<WesterosGameStat
         if (this.nightsWatchWon) {
             // Wildlings attack has been rebuffed
             // Check if there a single highest bidder or if there needs to be a decision from the iron throne holder
-            if (this.highestBidders.length > 1) {
+            // (Automatically resolve highest bidder for Silence At The Wall)
+            if (this.wildlingCard.type != silenceAtTheWall && this.highestBidders.length > 1) {
```

Comment:
This is not the best way to handle this as it is not generic, but I'll merge this and I'll create an issue to note this somewhere.